client/landing/gutenboarding/available-designs.ts

```diff
@@ -1,3 +1,14 @@
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from '../../config';
+import type { Design } from './stores/onboard/types';
+
 const availableDesigns: Readonly< AvailableDesigns > = {
```

Comment:
If `availableDesigns` moves to data layer (we'll pull them from an API soon enough), would there be another place that's convenient for these utils?

Reply:
I think we can figure that out then. Because now, it will be easier for whoever migrated to the APi to know where all the logic lives to move it around.